tests/behat/theme_boost_union_looksettings_mobile.feature

```diff
@@ -0,0 +1,20 @@
+@theme @theme_boost_union @theme_boost_union_looksettings @theme_boost_union_looksettings_mobile
+Feature: Configuring the theme_boost_union plugin for the "mobile" tab on the "Look" page
+  In order to use the features
+  As admin
+  I need to be able to configure the theme Boost Union plugin
+
+  @javascript
+  Scenario: mobilecss:  - Insert content in editor for additional css which is only displayed in the App.
```

Comment:
In addition to this scenario, two other scenarios might make sense:

1. A scenario which tests that a pre-set mobilecssurl is overwritten after the setting is saved.
2. A scenario which tests that the mobilecssurl is cleared after all CSS code is removed from the setting.